php/tests/encode_decode_test.php

```diff
@@ -95,6 +95,20 @@ public function testEncodeDecodeOneof()
         $n = new TestMessage();
         $n->mergeFromString($data);
         $this->assertSame("oneof_enum", $n->getMyOneof());
+
+        $m->setOneofString("");
+        $data = $m->serializeToString();
+        $n = new TestMessage();
+        $n->mergeFromString($data);
+        $this->assertSame("oneof_string", $n->getMyOneof());
```

Comment:
Is it possible to also have an assertion to test that the value is equal to `""` here? Or is that unnecessary?